v1/plugin/plugin_test.go

```diff
@@ -239,7 +239,7 @@ func TestMakeTLSConfig(t *testing.T) {
 		Convey("plugin lib should use TLS config requiring verified clients and specific cipher suites", func() {
 			config := tlsSetupInstance.makeTLSConfig()
 			So(config.ClientAuth, ShouldEqual, tls.RequireAndVerifyClientCert)
-			So(config.PreferServerCipherSuites, ShouldEqual, true)
+			//So(config.PreferServerCipherSuites, ShouldEqual, true) //nolint:SA1019
```

Comment:
staticcheck complained on this usage. there are no alternatives - hence commented